1386 content tweaks misc changes

SteveMilnes opened the pull request:
## Changes

Full stops added to project handover confirmation page. Also explicitly call out the URN to make it clearer in first sentence.
<img width="1047" alt="Screenshot 2023-03-28 at 15 59 59" src="https://user-images.githubusercontent.com/104517016/228280657-1477388f-9d9c-4460-925e-3f6b4c7fe10a.png">

## Checklist

- [x] Attach this pull request to the appropriate card in Trello.
- [x] Update the `CHANGELOG.md` if needed.

SteveMilnes commented:
On the Completed projects list is it possible that the table heading `Completed At` becomes `Completed on`?

lozette commented:
You still have a failing test, sorry - you need to update the expectation on https://github.com/DFE-Digital/dfe-complete-conversions-transfers-and-changes/blob/734c2ea46171fd5a1428ce30617fd219b755a16c/spec/features/users_can_create_voluntary_conversion_projects_spec.rb#L55 to include the new full stop

SteveMilnes commented:
Thanks @lozette. I've added a full stop on line 55. I also noticed there was a missing full stop on line 56 there too, so I added one. Hopefully they're in the right place. I figured the text inside the "" was the content to added the full stops inside those. Let me know if that's not right and I'll try again.
